NLibrary/Classes/ProfileMVVM/Prototypes/ProfileRepoProtocol.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Limit vertical whitespace to a single empty line. Currently 2.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5382838)